test/drop_test/error.py

```diff
@@ -36,9 +20,8 @@ class Error(IntEnum):
     DB_LOST = (32,)
     FILE_CHECKSUM_MISMATCH = (33,)
     FILE_REJECTED = (34,)
-    FILE_PAUSED = (35,)
-    FILE_FAILED = (36,)
-    FILE_FINISHED = (37,)
+    FILE_FAILED = (35,)
+    FILE_FINISHED = (36,)
```

Comment:
Changing numerical values of these breaks the API

Reply:
Yeah, I'm aware. These were introduced lately and never reached the main branch, thus I considered it safe to change

Reply:
Okay. Please update v5 docs as well